pkg/api/k8s_versions.go

```diff
@@ -350,49 +350,53 @@ func GetK8sComponentsByVersionMap(k *KubernetesConfig) map[string]map[string]str
 
 func getVersionOverridesMCR(v string) map[string]string {
 	switch v {
+	case "1.18.6":
+		return map[string]string{common.WindowsArtifactComponentName: "v1.18.6-hotfix.20200723/windowszip/v1.18.6-hotfix.20200723-1int.zip"}
 	case "1.18.4":
-		return map[string]string{common.WindowsArtifactComponentName: "v1.18.4-hotfix.20200624/windowszip/v1.18.4-hotfix.20200624-1int.zip"}
+		return map[string]string{common.WindowsArtifactComponentName: "v1.18.4-hotfix.20200626/windowszip/v1.18.4-hotfix.20200626-1int.zip"}
```

Comment:
The date for this patch is different the rest. Is that correct?

Reply:
v1.18.4-hotfix.20200626 was built with CVE-2020-8559 several weeks ago. I updated it since it is not the latest hotfix release.